src/main/java/com/marginallyclever/makelangelo/plotter/plotterRenderer/MakelangeloCustom.java

```diff
@@ -253,141 +213,9 @@ private void paintPenHolderToCounterweights( GL2 gl2, Plotter robot ) {
 		gl2.glEnd();
 	}
 
-	private void paintPlotter(GL2 gl2,float gx,float gy) {
+	private void paintPlotter(GL2 gl2, float gx, float gy) {
 		// plotter
 		gl2.glColor3f(0, 0, 1);
-		drawCircle(gl2,(float)gx,(float)gy,PEN_HOLDER_RADIUS_5);
+		drawCircle(gl2, gx, gy, PEN_HOLDER_RADIUS_5);
 	}
-	
-	private void drawCircle(GL2 gl2,float x,float y,float r) {
-		gl2.glTranslatef(x, y, 0);
-		gl2.glBegin(GL2.GL_LINE_LOOP);
-		float f;
-		for(f=0;f<2.0*Math.PI;f+=0.3f) {
-			gl2.glVertex2d(
-					Math.cos(f)*r,
-					Math.sin(f)*r);
-		}
-		gl2.glEnd();
-		gl2.glTranslatef(-x, -y, 0);
-	}
-	
-	private void drawArc(GL2 gl2,float x,float y,float r, float a1, float a2) {
-		gl2.glTranslatef(x, y, 0);
-		gl2.glBegin(GL2.GL_LINES);
-		float f;
-		int i;
-		int steps=10;
-		float delta=(a2-a1)/(float) steps;
-		f=a1;
-		for(i=0;i<steps;i++) {
-			gl2.glVertex2d(	Math.cos(f)*r,Math.sin(f)*r);
-			gl2.glVertex2d(	Math.cos(f+delta)*r,Math.sin(f+delta)*r);
-			f += delta;
-		}
-		gl2.glEnd();
-		gl2.glTranslatef(-x, -y, 0);
-	}
-	
-	@SuppressWarnings("unused")
-	private void paintSafeArea(GL2 gl2,Plotter robot) {
-		PlotterSettings settings = robot.getSettings();
-		double topcrit_ang=20*3.14/180.0;
-		double sidecrit_ang=20*3.14/180.0;
-		double top = settings.getLimitTop();
-		double bottom = settings.getLimitBottom();
-		double left = settings.getLimitLeft();
-		double right = settings.getLimitRight();
-		double top_crit=top-(right-left)/2.0*Math.tan(topcrit_ang);
-		double side_crit=(top-top_crit)*Math.tan(sidecrit_ang);
-		double belt=Math.sqrt(Math.pow(top-bottom,2)+Math.pow((right-left)/2.0f,2));
-		double bot_crit=top-Math.sqrt(Math.pow(top-bottom,2)-Math.pow(right-left,2)*0.75);
-		double botang=Math.asin((right-left)/(2*belt));
-		double sideang=Math.asin((right-left)/(belt));
-		
-		gl2.glColor4f(1f,0.4f,0.4f,0.75f);
-
-		gl2.glBegin(GL2.GL_LINES);
-
-		gl2.glVertex2d(left+side_crit, bot_crit);
-		gl2.glVertex2d(left, top);
-
-		gl2.glVertex2d(left, top);
-		gl2.glVertex2d((right+left)/2, top_crit);
-
-		gl2.glVertex2d((right+left)/2, top_crit);
-		gl2.glVertex2d(right, top);
-
-		gl2.glVertex2d(right, top);
-		gl2.glVertex2d(right-side_crit, bot_crit);
-
-		gl2.glEnd();
-		drawArc(gl2, (float)right, (float)top, (float)belt, (float)(Math.PI*1.5-sideang), (float)(Math.PI*1.5-botang));
-		drawArc(gl2, (float)left, (float)top, (float)belt, (float)(Math.PI*1.5+botang), (float)(Math.PI*1.5+sideang));
-	}
-/*
```

Comment:
i am keeping these until we have templates setup for each machine style.  
Then these number can be copied into the template.  Same is true for other machine types.

Reply:
ok I put this back